ts-apps/decision-logic-visualizer/src/lib/layout-ir/ladder-lir.svelte.ts

```diff
@@ -683,7 +748,7 @@ export const emptyBundlingNodeAnno: BundlingNodeAnno = { annotation: '' }
 export function isBundlingFlowLirNode(
   node: LadderLirNode
 ): node is BundlingFlowLirNode {
-  return isSourceLirNode(node) || node instanceof SinkLirNode
+  return isSourceLirNode(node) || isSinkLirNode(node)
```

Comment:
This and the other changes in the file below is just code cleanups / refactoring that are not related to the main point of the PR. Sorry for not having put it in another PR.